Tests/MongoSwiftTests/DNSSeedlistTests.swift

```diff
@@ -33,10 +33,6 @@ struct DNSSeedlistTestCase: Decodable {
 }
 
 final class DNSSeedlistTests: MongoSwiftTestCase {
-    override func setUp() {
```

Comment:
to take a step back: I think a general principle we try to adhere to in our work is keeping the code changes in a pull request scoped to the task at hand. this makes it easier to track the history of code over time and reason about why particular changes were made, determine why a test started failing or became flaky at some point, etc. it also makes code review easier since there is less to review and it is easier for reviewers to understand the changes. 

this file contains tests that do not use the unified test runner code in any way, so I do not think making any edits to this file would be considered in scope for this ticket.

from a technical perspective, this change _may_ be fine to make, but I'd rather avoid including unnecessary changes for the reasons I note above. you can see what the property does here: https://developer.apple.com/documentation/xctest/xctestcase/1496260-continueafterfailure. in the past we would use `continueAfterFailure` in tests where it was likely that if one assertion failed every assertion following it would also fail. we decided to stop relying on this in new tests at some point because it didn't work properly on Linux, which is why you likely haven't seen it in other files. I'm not sure if that is still the case re Linux, but at least on macOS it provides some value in cases where a test newly starts failing. in cases where tests are passing you wouldn't see any difference from this being present, but for a failing test it can affect the outcome.

Reply:
Oh yep I think my incorrect assumptions about the purpose of the method led me to believe that it was in scope with unified test runner as part of the conversions. Makes sense that we shouldnt alter this if it isnt actually in scope and will revert this change here. Also thanks for the link, the info helps contextualize a bit.  

All changes related to `setUp()` have been reverted. Heres the other places where the diff takes place and a quick summary of them

`CommandMonitoringTests` - only 1 test method that uses the unified runner. 

`MongoCursorTests` - same situation as DNSSeedlist where no unified tests are run.

`ReadPreferenceTests` - same as above.

`UnifiedTests` - uses the unified runner but also has normal prose tests.

Maybe we can open a ticket to re-assess the need for this method (esp in a file in `CommandMonitoringTests.swift`)?